### DISCUSSION TITLE
Latest deepmd-kit version needs Tensorflow 2.x?

### DISCUSSION THREAD

**Discussion by rohskopf** [deepmodeling/deepmd-kit#199] [Q&A] [answered]
Whenever I do `pip install deepmd-kit`, and run the `dp -h` test, it fails if unless my Tensorflow version is 2.x. 

The error is:
`tensorflow.python.framework.errors_impl.NotFoundError: libtensorflow_framework.so.2: cannot open shared object file: No such file or directory`

So if you use Tensorflow version 1.x with the latest deepmd-kit, it won't work?

Just wondering if this is a bug or if this is intentional.

**Comment by njzjz** [comment, marked as answer]
We compiled the pypi package with the newest TensorFlow (2.1). You can compile deepmd-kit with any TensorFlow version, just using

pip install git+https://github.com/deepmodeling/deepmd-kit

Thanks!

**Reply by wanghaopku** [reply]
I don't knows the difference between the two method of install the deepmd-kit
1、git clone https://github.com/deepmodeling/deepmd-kit.git
     then "pip install ."
     i tried this method, it reports a mistake "ERROR: Could not build wheels for deepmd-kit which use PEP 517 and cannot be installed directly".  my pip verison is 20.0.1
2、pip install git+https://github.com/deepmodeling/deepmd-kit
      Then i  used the method you mentioned  here. The installation is successful.

I wonder what is the difference....

**Reply by njzjz** [reply]
@wanghaopku you may post the full log of `pip install .`. `Could not build wheels` is just a general error and cannot show any information.
